tenable/io/v3/base/schema/export/filters.py

```diff
@@ -0,0 +1,106 @@
+"""
+Base Universal Workspace Filter Schema
+"""
+from marshmallow import Schema, ValidationError, fields, pre_load
+
+
+class FilterSchema(Schema):
+    """
+    Schema supporting both the Filter and FilterGroups
+    """
+
+    property = fields.Str()
+    operator = fields.Str()
+    value = fields.Raw()
+    and_ = fields.List(fields.Nested("FilterSchema"), data_key="and")
+    or_ = fields.List(fields.Nested("FilterSchema"), data_key="or")
+
+    @pre_load(pass_many=False)
+    def validate_and_transform(self, data, **kwargs):  # noqa: PLW0613
+        """
+        Handles schema validation and data transform based on the data
+        presented.
+        """
+        if (  # noqa: PLR1705
```

Comment:
Might read better if you pull these conditions out into their own methods

Reply:
I personally like to encapsulate complex logic like that in a separate method that can provide the evaluation, but I'm ok with adding comments to describe and refactoring later if we want

Reply:
Added the comments in the code for filter.py